### PR TITLE
Docs contribution update

### PR DESCRIPTION
**Pull request by qduk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,6 +150,38 @@ The project is following Network to Code software development guidelines and are
 - Black, Pylint, Bandit, flake8, and pydocstyle for Python linting and formatting.
 - pytest, coverage, and unittest for unit tests.
 
+There are a number of things that are required in order to have a successfull PR.
+
+- All new functions must contain at least 1 example in their docstrings.
+- Docstrings must conform to the google docstring standard.
```
Comment (jeffkala) [inline README.md:156]:
```suggestion
- Docstrings must conform to the google docstring [convention](https://google.github.io/styleguide/pyguide.html#381-docstrings).
```

File: tests/unit/test_docs.py [hunk 2]
```diff
@@ -48,7 +48,7 @@
         "name": "contribution",
         "start_line": 145,
         "start_value": "Pull requests are welcomed and automatically built and tested against multiple versions of Python through TravisCI.\n",
```
Comment (jeffkala) [inline tests/unit/test_docs.py:50]:
```suggestion
        "start_value": "Pull requests are welcomed and automatically built and tested against multiple versions of Python through GitHub Actions.\n",
```

**Review by jeffkala — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -150,6 +150,38 @@ The project is following Network to Code software development guidelines and are
...
+- Unit test for newly added functions are required.
+- If applicable, tests related to config parsing and compliuance must be added.
+- Update the jinja2 filter for any new functions.
```
Comment (itdependsnetworks) [inline README.md:159]: Can you explain this a bit more?

Comment (itdependsnetworks) [reply README.md:159]: nvm, can you just add a `(see below for details)` to the end of the doc

**Review by itdependsnetworks — APPROVED** [review]

**Pull request merged**